towhee/functional/mixins/milvus_mixin.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from towhee.hparam import param_scope
+from ..entity import Entity
+
+
+def insert(self, index, collection, batch_size):
```

Comment:
it's better to tell what the function does by its name, `_milvus_insert` should be better.

Reply:
and the first argument of this function should be iterable, `self` should only be used for instance methods/